.circleci/config.yml

```diff
@@ -9,6 +9,7 @@ jobs:
       - run: sudo apt-get update && sudo apt-get install -y build-essential libfuse-dev libsqlite3-dev libgmp-dev libncurses5-dev pkg-config git g++ gcc libre2-dev python3-sphinx
       - run: make tarball
       - run: make test
+      - run: make unittest
```

Comment:
This runs the tests on ubuntu 20.04, is there value in running them within the other containers?

Reply:
At the moment no. This is intended algorithmic code not code that depends on details of the operating system, just the architecture and glibc really. We might read form files in these unit tests but that's about it. I don't anticipate that changing for a long time if ever. We can make a separate OS verification test suite later (might have to for the blob store alone)

Reply:
For that I think I might duplicate all the OSs here and then just rebuild wake on them instead of trying to use the package stuff since I don't want to package unit tests.